auxx/src/Repl.hs

```diff
@@ -89,10 +88,10 @@ createAuxxRepl = do
     lastResultVar <- newEmptyMVar
     (printActionVar :: MVar (PrintAction IO)) <- newEmptyMVar
     let
-        withCommand :: forall m. (MonadIO m, Mockable Catch m) => (Text -> m ()) -> m ()
+        withCommand :: forall m. (MonadIO m, MonadCatch m) => (Text -> m ()) -> m ()
         withCommand cont = do
             cmd <- takeMVar nextCommandVar
-            res <- handle
+            res <- handleAsync
```

Comment:
Is it intentional or you just want to preserve existing logic and change it to `handleAny` later?
_perhaps I can figure that out by reading the code, but I don't want to ;d_

Reply:
It is intentional: when a command is interrupted with an async exception, we do not want the REPL to crash.